adapter.go

```diff
@@ -92,6 +92,18 @@ func NewAdapterWithKey(network string, address string, key string) (*Adapter, er
 	return newAdapter(network, address, key, "", "")
 }
 
+// NewAdapterWithPool is the constructor for Adapter.
+func NewAdapterWithPool(pool *redis.Pool) (*Adapter, error) {
```

Comment:
Also add `NewAdapterWithPool` to README.md as an example

Reply:
Ok, done. Sorry, I didn't noticed that